modules/setting-up-cpu-manager.adoc

```diff
@@ -101,6 +99,13 @@ cpuManagerReconcilePeriod: 5s   <2>
 <1> `cpuManagerPolicy` is defined when you create the `KubeletConfig` CR.
 <2> `cpuManagerReconcilePeriod` is defined when you create the `KubeletConfig` CR.
 
+. Create a project, for example, `cpuman` by running the following command:
++
+[source,terminal]
+----
+$ oc new-project cpuman
+----
```

Comment:
If the name `cpuman` isn't critical elsewhere for examples:

```suggestion
. Create a project by running the following command:
+
[source,terminal, subs="normal"]
----
$ oc new-project _<project_name>_
----
```

Reply:
@maxwelldb What does `subs="normal"` do? Couldn't find it anywhere in the repo

Reply:
@SNiemann15 It appears that `subs="normal"` allows markup in the code block? Max's example adds italic. The first example here uses `subs="normal"`, the second one does not. 

![image](https://github.com/openshift/openshift-docs/assets/25209155/a5b7c1e6-ecf3-4850-acf2-19970b6fa22c)

Asciidoctor docs, [The subs attribute](https://docs.asciidoctor.org/asciidoc/latest/subs/apply-subs-to-blocks/#the-subs-attribute)

Reply:
I think I'll keep it without italic. Checked other instances of `oc new-project <....` and also other commands didn't find any variables in codeblocks that are italic.

Reply:
@SNiemann15 Yeah, my bad. Red Hat as a whole does use italics for user replaceable values in code, but openshift-docs does not. It's one of those house style things. :shrug: